mne/preprocessing/eog.py

```diff
@@ -49,15 +49,19 @@ def find_eog_events(raw, event_id=998, l_freq=1, h_freq=10,
     eog_inds = _get_eog_channel_index(ch_name, raw)
     logger.info('EOG channel index for this subject is: %s' % eog_inds)
 
-    eog, _ = raw[eog_inds, :]
+    # Reject bad segments.
+    eog, times = raw.get_data(picks=eog_inds, reject_by_annotation='omit',
```

Comment:
this is technically an API change, so we should probably expose a `reject_by_annotation` param in `find_eog_events`